controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -174,6 +183,48 @@ func (r *OCSInitializationReconciler) Reconcile(ctx context.Context, request rec
 		r.Log.Error(err, "Failed to ensure uxbackend service")
 		return reconcile.Result{}, err
 	}
+	isROSAHCP, err := platform.IsPlatformROSAHCP()
+	if err != nil {
+		r.Log.Error(err, "Failed to determine if on ROSA HCP platform")
+	}
+	if isROSAHCP {
```

Comment:
if you hit error this'll always be false and returning here also wouldn't work, so ya you need to know for sure whether the platform is rosa or not.

Reply:
On getting error, reconciling again

Reply:
Check [comment](https://github.com/red-hat-storage/ocs-operator/pull/2465#discussion_r1535274622)